### DISCUSSION TITLE
是否有支持qq频道的计划？

### DISCUSSION THREAD

**Discussion by Wind2009-Louse** [nonebot/discussions#77] [问答] [answered]
go-cqhttp已经提供qq频道相关的API，请问最近是否有计划更新相关的支持内容？或者说要等到QQ频道正式公测的时候才会提供相关功能？

**Comment by yanyongyu** [comment, marked as answer]
目前没有也不会针对gocq做专门的处理，仅做了onebot标准，不过你可以通过custom model来实现

QQ频道目前有官方API，已在 https://github.com/nonebot/adapter-qqguild 进行编写
